README.md

```diff
@@ -29,116 +28,60 @@ and `CONTRIBUTING_HSE.md` in the repo root directory.
-
+* `v2` contains the HSE `2.x` project documentation
+* `v1` contains the HSE `1.x` project documentation
 
 ## Site Generation Tools
 
 The HSE project documentation is based on
-[Material for MkDocs](https://squidfunk.github.io/mkdocs-material/),
-which is a theme for the [MkDocs](https://www.mkdocs.org/)
-static site generator.
-For versioning we use [Mike](https://github.com/jimporter/mike) which
-is natively integrated with Material for MkDocs.
-These tools are Python packages.
-
-The HSE documentation includes an API reference that is built from
-header files in the `hse` repo.  This process uses
-[doxygen](https://www.doxygen.nl/index.html) to generate XML that is converted
-to markdown using [doxybook2](https://github.com/matusnovak/doxybook2).
-
-To avoid having to install `doxygen`, `doxybook2`, and related dependencies,
-proxy API files are provided.  These enable the HSE project documentation
-to be edited and tested, including links to the API reference pages, without
-having to build the actual API documentation.
+[Material for MkDocs](https://squidfunk.github.io/mkdocs-material/), which is a
+theme for the [MkDocs](https://www.mkdocs.org/) static site generator. For
+versioning we use [Mike](https://github.com/jimporter/mike) which is natively
+integrated with Material for MkDocs. These tools are Python packages. If you
+need to get `mkdocs` and `mike` to build the site, use the `requirements.txt`
+file:
 
-
-## Building and Viewing
-
-Build the latest version of the complete HSE documentation as follows.
-In this example we are using the proxy API files.
-
-Clone the repo and install the site generation tools.
+Note that you will need Python `>= 3.7` to build the website.
 
 ```shell
-git clone https://github.com/hse-project/hse-project.github.io.git
-cd hse-project.github.io
-python3 -m venv venv
-source venv/bin/activate
 python3 -m pip install -r requirements.txt
 ```
 
-Copy the proxy API files to the required location.
-We do this because `.gitignore` includes `v2/docs/api` so as to ignore
-these files when built from the `hse` repo headers.
-
-```shell
-mkdir v2/docs/api
-cp -r v2/api-proxy/* v2/docs/api
-```
+The HSE documentation includes an API reference that is built from
+header files in the `hse` repo.  This process uses
+[doxygen](https://www.doxygen.nl/index.html) to generate XML that is converted
+to markdown using [doxybook2](https://github.com/matusnovak/doxybook2).
 
-Generate the site documentation.
+## Building, Viewing and Editing
 
-```shell
-mike deploy -F v1/mkdocs.yml 1.x
-mike deploy -F v2/mkdocs.yml 2.x
-mike set-default -F v2/mkdocs.yml 2.x
-```
+The website is built using [Meson](https://mesonbuild.com/).
 
-This build process creates a local branch named `gh-pages` with the
-generated static site files.  You can view these as follows.
+In order to build the documentation, run the following commands:
 
 ```shell
-mike serve -F v2/mkdocs.yml
+meson build
```

Comment:
Document the -Dapis=none option.  Also, in an appropriate place include information on dependencies for building doxybook2.